app/javascript/packages/build-sass/CHANGELOG.md

```diff
@@ -0,0 +1,3 @@
+## 1.0.0
```

Comment:
should/could we add a README too that describes how to use it? or is that in a separate PR?

Reply:
Ooh, I totally missed that this didn't have one yet. I'll definitely plan to add it.

Reply:
Added a README.md in ea264ff .